pkg/serializer/split/split.go

```diff
@@ -163,3 +169,14 @@ func marshal(m marshaler.Marshaler, mType MarshalType) ([]byte, error) {
 		return m.MarshalJSON()
 	}
 }
+
+func handleDroppedPayloads(droppedPayloads ...[]marshaler.Marshaler) {
+	log.Warnf("Payloads could not be splitted and were dropped")
+
+	var dropped int64
+	for _, payloads := range droppedPayloads {
+		dropped += int64(len(payloads))
```

Comment:
I am not convinced that the value reported by this metric is usable. When this line gets executed it means that we were not able to successfully split the payloads so the number of payloads does not have a real meaning. 
Ideas we could implement instead:
- record the number of occurrences - will ease the logic
- record the number of metrics dropped - will need changes to the marshaller interface so I am not sure

Reply:
Okay I agree, I didn't remember that a payload could contain several metrics when I wrote this. Let's just record the number of occurences then.